extensions/solana-explorer/package.json

```
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "solana-explorer",
+  "title": "Solana Explorer",
+  "description": "Look up transactions and accounts on the various Solana clusters.",
+  "icon": "command-icon.png",
+  "author": "darklong",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "solana-explorer",
+      "title": "Solana Explorer",
```

```suggestion
      "title": "Search Blockchain",
      "subtitle": "Solana Explorer",
```
This could also improve the UX, what do you think?